### ISSUE TITLE
Equipped items drop on rejoin.

### ISSUE THREAD

**Issue by DigiDK** [Extegral/Enigmatic-Legacy#501]
Mods: cealus-forge-3.2.0+1.20.1.jar curios-forge.5.9.0+1.20.1.jar EnigmaticLegacy-2.29.0.jar Patchouli-1.20.1-84-FORGE.jar
Recreation:
1. Create new world (creative, cheats on, peaceful)
2. Equip cursed ring, scroll of a thousand curses, enchanters pearl
3. rejoin
then the scroll and pearl should be in your hotbar again

**Comment by DarkScorn** [comment]
Together with those, Emblem of Bloodstained Valor unequips as well, on Forge 1.18.2 40.2.21, Enigmatic Legacy 1.18.2, 2.25.0, Curios 5.0.9.2, Caelus 3.0.0.2.

**Comment by FranFkntastic** [comment]
Also experiencing this in 1.20.1

**Comment by Seveska** [comment]
Experiencing this with Pact of Infinite Avarice scroll, on a server running Cisco's Fantasy Modpack v4d, mc 1.19.2, Forge 43.3.5, Enigmatic Legacy 2.26.5. Item returns to inventory on relog, or drops on ground if inventory is full.

**Comment by CwumbTheCrumb** [comment]
yep, expiriencing the same in 1.20.1, now Testament of Contempt included

**Comment by Exovi** [comment]
Also seeing this - the server I'm on does not have recipes for these items just a one-time reward from questbook.... lost my Pact of Infinite Avarice and I'm just stuck

**Comment by wizeshi** [comment]
Yeah, year and a bit later and the issue persists. Tried to use EL with Relics, but every time I enter the world a gazillion charms drop because of this (btw it has to be EL's fault, only occurs when EL charms are equipped).

**Comment by Exovi** [comment]
Only seems to happen with Seven Curses items

**Comment by cyberpwnn** [comment]
After taking a look at the source i figured out enigmatic was giving garbage data to curiosapi causing invalid slot and curios pops it off into your inv. I fixed it on my fork fir 1.20.1 forge, i also uploaded the jar so no one has to build it https://github.com/AuramMods/Enigmatic-Legacy/blob/1.20.X/EnigmaticLegacy-2.30.1.jar

Fix is https://github.com/Aizistral-Studios/Enigmatic-Legacy/commit/0df4866d61344070025abe18ebd6db1c639bc16d
